packages/components/src/test/instance-filter-builder/PresentationInstanceFilterDialog.test.tsx

```diff
@@ -74,7 +74,7 @@ describe("PresentationInstanceFilterDialog", () => {
   it("invokes 'onApply' with string property filter rule", async () => {
     const spy = sinon.spy();
     const { container, getByText, queryByDisplayValue, user } = render(
-      <PresentationInstanceFilterDialog imodel={imodel} descriptor={descriptor} onClose={() => {}} onApply={spy} isOpen={true} />,
+      <PresentationInstanceFilterDialog imodel={imodel} descriptor={descriptor} onClose={() => {}} onReset={() => {}} onApply={spy} isOpen={true} />,
```

Comment:
`onClose` and `onReset` can be removed from tests where `() => {}` is passed to reduce noise.